nvflare/fuel/utils/class_utils.py

```diff
@@ -80,24 +72,36 @@ def __init__(self, base_pkgs: List[str], module_names: List[str], exclude_libs=T
         self.base_pkgs = base_pkgs
         self.module_names = module_names
         self.exclude_libs = exclude_libs
+
+        self._logger = logging.getLogger(self.__class__.__name__)
         self._class_table = {}
         self._create_classes_table()
 
     def _create_classes_table(self):
         for base in self.base_pkgs:
             package = importlib.import_module(base)
 
-            for importer, modname, ispkg in pkgutil.walk_packages(path=package.__path__, prefix=package.__name__ + "."):
-
-                if modname.startswith(base):
-                    if not self.exclude_libs or (".libs" not in modname):
-                        if any(modname.startswith(base + "." + name + ".") for name in self.module_names):
+            for module_info in pkgutil.walk_packages(path=package.__path__, prefix=package.__name__ + "."):
+                module_name = module_info.name
+                if any(module_name.startswith(deprecated_package) for deprecated_package in DEPRECATED_PACKAGES):
```

Comment:
Why added the DEPRECATED_PACKAGES? This will result component initialization could not find the module. For example:  PTFileModelLocator.

Reply:
Because we have moved pt module from app common to app opt in PR #1348